documentation/3.4.0/dockerized/envs/interface_env

```diff
@@ -0,0 +1,26 @@
+API_DOC_URL=/interface/api-doc
+APPLICATION_ENV=development
+APPLICATION_PORT=3569
+ELEVATE_NOTIFICATION_KAFKA_BROKERS=10.148.0.38:9092
+ELEVATE_NOTIFICATION_KAFKA_GROUP_ID=dev.mentoring
+ELEVATE_NOTIFICATION_KAFKA_TOPIC=dev.notification
+ENTITY_SERVICE_BASE_URL=http://localhost:5001
+INSTALLED_PACKAGES=elevate-mentoring elevate-survey-observation@1.0.1 elevate-self-creation-portal
+MENTORING_SERVICE_BASE_URL=http://localhost:7101
+NOTIFICATION_SERVICE_BASE_URL=http://localhost:7201
+PROJECT_SERVICE_BASE_URL=http://localhost:6000
+RATE_LIMITER_ENABLED=true
+RATE_LIMITER_GENERAL_LIMIT=500
+RATE_LIMITER_NUMBER_OF_PROXIES=3
+RATE_LIMITER_PUBLIC_LOW_LIMIT=5
+REQUIRED_BASE_PACKAGES=elevate-mentoring elevate-project elevate-survey-observation self-creation-portal
+REQUIRED_PACKAGES=elevate-mentoring@1.2.90 elevate-survey-observation@1.0.22 elevate-project@1.1.53 shiksha-notification@1.1.3 elevate-self-creation-portal@1.0.70
+ROUTE_CONFIG_JSON_URLS_PATHS=https://raw.githubusercontent.com/ELEVATE-Project/utils/refs/heads/develop/interface-routes/elevate-routes.json
+SAAS_NOTIFICATION_BASE_URL=interface/v1/notification/send-raw
```

Comment:
@MallanagoudaB Can u check with @VISHNUDAS-tunerlabs or @rakeshSgr that we need to use our notification service instead of saas